libheif/CMakeLists.txt

```diff
@@ -133,12 +134,22 @@ else ()
     message("Not compiling 'libsharpyuv'")
 endif ()
 
-if (WITH_DEFLATE_HEADER_COMPRESSION)
+if (WITH_DEFLATE_HEADER_COMPRESSION OR WITH_UNCOMPRESSED_CODEC)
     find_package(ZLIB REQUIRED)
     target_link_libraries(heif PRIVATE ZLIB::ZLIB)
+    target_compile_definitions(heif PRIVATE WITH_ZLIB_COMPRESSION=1)
```

Comment:
Could we rename this to `HAVE_ZLIB` to be consistent with `HAVE_BROTLI` ?
We could even make zlib optional like brotli.

Reply:
Fixed.